components/collective-page/hero/HeroBackground.js

```diff
@@ -30,9 +31,9 @@ const Cropper = dynamic(() => import(/* webpackChunkName: 'react-easy-crop' */ '
 });
 
 // Dynamically import dropzone component
-const Dropzone = dynamic(() => import(/* webpackChunkName: 'react-dropzone' */ 'react-dropzone'), {
-  ssr: false,
-});
+// const { useDropzone } = dynamic(() => import(/* webpackChunkName: 'react-dropzone' */ 'react-dropzone'), {
```

Comment:
`react-dropzone` should stays imported dynamically here (as it was before the update)